EventsExpress/ClientApp/src/store/initialState.js

```diff
@@ -36,7 +36,11 @@ const initialState = {
         isRegisterSuccess: false,
         registerError: null
     },
-    change_avatar: {
+    unitsOfMeasuring: {
+        isPending: true,
+        isError: false,
+        units: []
+    },    change_avatar: {
```

Comment:
Starte new piece of state at the new line.

Reply:
Solved